tests/integration/kubernetes/tests_common.sh

```diff
@@ -38,6 +38,29 @@ get_pod_config_dir() {
 	info "k8s configured to use runtimeclass"
 }
 
+get_tee() {
+	tees[qemu-tdx]="tdx"
+
+	tee=${tees[${KATA_HYPERVISOR}]}
+}
+
+get_tee_key_resource() {
+	tees_key_resource[qemu-tdx]="tdx.intel.io/keys"
+
+	tee_key_resource=${tees_key_resource[${KATA_HYPERVISOR}]}
+}
+
+get_tee_key_label_jsonpath() {
+	# Sadly we need to declare the whole `-o jsonpath={...}`, as I couldn't find a reliable
+	# way to just set the TEE specific capacity string in a way I could pass it as an env
+	# var to the -o jsonpath option.
+	#
+	# We need to escape the dots in order to make jsonpath happy
+	tees_key_label_jsonpath[qemu-tdx]="-o jsonpath='{.status.capacity.tdx\.intel\.io/keys}'"
```

Comment:
Is that because tdx.intel.io is itself a key and not a path?
Or did you need to escape all of those dots?

Reply:
We need to escape the dots because for each **item** in capacity, the json path would be something like `.status.capacity.item`.  Any other `.` there would indicate a subfield, unless escaped, which them indicates an item name containing dots.